frontend/src/components/pages/GlobalForms/GlobalFormsPage.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {Text, Container} from '@chakra-ui/react'
+import AdminAPIClient from "../../../APIClients/AdminAPIClient";
+import { ClauseInfo } from "../../../types/AdminTypes";
+import WaiversTab from "./WaiversTab";
+
+const GlobalFormsPage = (): React.ReactElement => {
+
+  const [waiver, setWaiver] = React.useState([]as ClauseInfo[]);
+  React.useEffect(() => {
+    const getWaiver = async () => {
+      const res = await AdminAPIClient.getWaiver();
```

Comment:
If you define the return type in the getWaiver function, then you'll know the type of the res variable here. That way you can check if you got a proper waiver back. A good example is on the camp overview `index.ts` component which makes a get request to retrieve all the info for that particular camp:

```
useEffect(() => {
    const getCamp = async () => {
      const campResponse = await CampsAPIClient.getCampById(campId);
      if (campResponse) {
        setCamp(campResponse);
      }
    };
    getCamp();
  }, [campId]);
```